### ISSUE TITLE
Too many filesystem checks error is encountered when using no-op rewrite on Vercel

### ISSUE THREAD

**Issue by ijjk** [vercel/next.js#14738]
When leveraging the custom-routes-proxying example, the no-op rewrite causes requests for some static assets to respond with a 502 error from too many filesystem checks ([example asset](https://custom-routes-49pcxcvhc.vercel.app/docs/_next/static/runtime/main-a4b218b9dd123f0be1fe.js)). This does not occur locally and only after deploying

This appears to be related to the proxy and not Next.js specifically cc @dav-is

**Comment by ijjk** [comment]
Closing as this has been resolved

**Comment by ahmedB7r** [comment]
I have the same problem here 
[https://store-alpha.vercel.app/admin/signin](url)
I have create-react-app build and want to serve it in my nextjs app
so I make just one rewrite in my next.config.js
```
  rewrites: async () => [
    {
      source: "/admin/:page*",
      destination: "/api/static/admin/:page*",
    },
  ],
```
and worked prefectly local
and get this error with vercel

**Comment by ahmedB7r** [comment]
working normally with heroku !!!
https://store-client-b7r.herokuapp.com/admin/signin

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
